src/Components/CountdownTimer/CountdownTimer.js

```diff
@@ -15,17 +16,25 @@ import resetTimerIcon from '../../assets/timer/resetTimerIcon.png'
 const CountdownTimer = () => {
   const [ settings ] = useContext(SettingsContext)
   const [ view, setView ] = useContext(ViewContext)
+  const [ session, setSession ] = useContext(SessionContext)
   
+  const recordFocusInterval = (interval) => {
```

Comment:
Glad this combination of resetting the session & setting the new focus interval works - still unsure exactly why we weren't able to do 2 separate setSessions!

Reply:
Same, I think that this is a great workaround, but it definitely makes me want to do some more thorough study of Hooks!